### ISSUE TITLE
"ø" and "Ø" characters in track title make the app crash

### ISSUE THREAD

**Issue by Slater91** [bleskodev/rubyripper#604]
Hi there! I found out that "ø" and "Ø" chars make Ruby Ripper crash; I am fairly sure about that because when removing the chars from the titles, the app works flawlessly. I am using the 2.3.1p112 version on Linux Mint 18.1 x64. I attached the terminal output.

[rubyripper-output.txt](https://github.com/bleskodev/rubyripper/files/706928/rubyripper-output.txt)

I do not know whether this is a bug in Ruby Ripper or in Ruby, so please forgive me if the latter is correct. Please ask if you need any further information.

Thank you! :)

**Comment by Safari77** [comment]
very funny! If I add char ```”``` ```("\0342\0200\0235")```  into album title, I get:

```
Not enough disk space left! Rip aborted
```
```
DEBUG: Free disk space is 1 MB
```

In reality I have 71 GiB free.
When I remove the utf8 quote char, ripping works.

BTW. are you using UTF-8 locale? E.g. ```it_IT.utf8```

**Comment by Slater91** [comment]
Yes, I am. By the way, I found out it crashes when I use the char "ö", too!

Il 26/01/2017 13:54, Sami Farin ha scritto:

very funny! If I add char ” ("\0342\0200\0235") into album title, I get:

Not enough disk space left! Rip aborted


DEBUG: Free disk space is 1 MB


In reality I have 71 GiB free.
When I remove the utf8 quote char, ripping works.

BTW. are you using UTF-8 locale? E.g. it_IT.utf8

**Comment by Safari77** [comment]
```ö``` does not crash for me, I have Fedora 24 with glib2-2.50.2, icu-56.1, ruby-2.3.3-61.1.fc24. glibc-2.23.1-11.fc24, rubyripper latest git.

**Comment by bleskodev** [comment]
Hi! Thanks for reporting this and sorry for the inconvenience.

@Slater91: I've searched the (imported) bug history, bugs such as this one already existed in previous versions... some seem to be fixed some not. 
I have tested this with the latest github version and I don't have this problem (neither with "ø", "Ø" or ö). Which version of rubyripper are you using? If you are not using the latest version from github, would it be possible for you to try with that one?

Thanks.
